browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -4354,7 +4404,46 @@ index 46169f509995747f4df200c3c08674c7735a875a..a3ee4b0e4804480f50cf2d05d528bd11
  #include "Logging.h"
  #include <wtf/CompletionHandler.h>
  
-@@ -257,26 +258,32 @@ void PolicyChecker::checkNewWindowPolicy(NavigationAction&& navigationAction, Re
+@@ -203,10 +204,12 @@ void PolicyChecker::checkNavigationPolicy(ResourceRequest&& request, const Resou
+     auto requestIdentifier = PolicyCheckIdentifier::create();
+     m_delegateIsDecidingNavigationPolicy = true;
+     String suggestedFilename = action.downloadAttribute().isEmpty() ? nullAtom() : action.downloadAttribute();
++    InspectorInstrumentation::willCheckNavigationPolicy(m_frame, request.url());
+     FramePolicyFunction decisionHandler = [this, function = WTFMove(function), request = ResourceRequest(request), formState = std::exchange(formState, nullptr), suggestedFilename = WTFMove(suggestedFilename),
+          blobURLLifetimeExtension = WTFMove(blobURLLifetimeExtension), requestIdentifier, isInitialEmptyDocumentLoad] (PolicyAction policyAction, PolicyCheckIdentifier responseIdentifier) mutable {
+         if (!responseIdentifier.isValidFor(requestIdentifier)) {
+             RELEASE_LOG_IF_ALLOWED("checkNavigationPolicy: ignoring because response is not valid for request");
++            InspectorInstrumentation::didCheckNavigationPolicy(m_frame, false);
+             return function({ }, nullptr, NavigationPolicyDecision::IgnoreLoad);
+         }
+ 
+@@ -219,21 +222,25 @@ void PolicyChecker::checkNavigationPolicy(ResourceRequest&& request, const Resou
+             FALLTHROUGH;
+         case PolicyAction::Ignore:
+             RELEASE_LOG_IF_ALLOWED("checkNavigationPolicy: ignoring because policyAction from dispatchDecidePolicyForNavigationAction is Ignore");
++            InspectorInstrumentation::didCheckNavigationPolicy(m_frame, false);
```

Comment:
This is misleading as in case of PSON old page receives 'Ignore' but the navigation continues in the new process. Are we ok with that?